src/legacy/core_plugins/data/public/search/fetch/fetch_soon.test.ts

```diff
@@ -72,22 +72,6 @@ describe('fetchSoon', () => {
     expect(callClient).toBeCalled();
   });
 
-  test('should delay by 50ms if config is set to batch searches', () => {
-    const config = getConfigStub({
-      'courier:batchSearches': true,
-    });
-    const request = {};
-    const options = {};
-
-    fetchSoon(request, options, { config } as FetchHandlers);
-
-    expect(callClient).not.toBeCalled();
-    jest.advanceTimersByTime(0);
-    expect(callClient).not.toBeCalled();
-    jest.advanceTimersByTime(50);
-    expect(callClient).toBeCalled();
-  });
-
   test('should send a batch of requests to callClient', () => {
     const config = getConfigStub({
       'courier:batchSearches': true,
```

Comment:
If we decide to totally get rid of batchSearches param, this can go too (and below) and in ui_setting_default.